WebSuggest/websuggest.ini

```diff
@@ -37,6 +37,12 @@
 # * Default: yes
 #enable_predefined_items = yes
 
+# Time that the plugin will wait before sending the request.
+# * Time in seconds (can be used with float type)
+# * The minimum value is 0.25

```

Comment:
Please specify a hard-coded maximum value to prevent mistyping. 3 seconds should do it I think